lib/python/qmk/cli/console.py

```diff
@@ -0,0 +1,302 @@
+"""Acquire debugging information from usb hid devices
+
+cli implementation of https://www.pjrc.com/teensy/hid_listen.html
+"""
+from pathlib import Path
+from threading import Thread
+from time import sleep, strftime
+
+import hid
+import usb.core
```

Comment:
This introduces a hard dependency on the `hid` and `pyusb` packages .
Without them, the `qmk` command can not be used at all anymore.
Was this intentional?

Reply:
Fix submitted in https://github.com/qmk/qmk_firmware/pull/12978

Reply:
Yes, that was intentional, those dependencies are now in `requirements.txt` and the necessary OS packages are in the install scripts.